contents/tutorials/vue-analytics.md

```diff
@@ -189,84 +184,15 @@ createApp(App)
 .mount('#app');
 ```
 
-Once you’ve done this, reload your app and click the buttons a few times. You should see events appearing in the [PostHog events explorer](https://app.posthog.com/events).
+Once you’ve done this, reload your app and click the buttons a few times. You should see events and pageviews appearing in the [activity explorer](https://us.posthog.com/activity/explore).
```

Comment:
> [activity explorer](https://us.posthog.com/activity/explore).

Should this be `[activity explorer](https://app.posthog.com/activity/explore).`? What's the difference?

Reply:
1. https://app.posthog.com/ is the old https://us.posthog.com/
2. Some people think https://app.posthog.com/ is the only URL that redirects to either `us` or `eu` depending if you're logged in or not, but https://us.posthog.com/ does too so it's fine